proposals/Alternative Schema/alternative_schema_examples.md

```diff
@@ -0,0 +1,53 @@
+## Change: Add Alternative Schema Examples
+
+The following text is to be inserted after the Alternative Schema Object section.
+
+### Alternative Schema Examples
+
+Minimalist usage of alternative schema:
+
+    schema:
+        x-oas-draft-alternativeSchema:
+          type: jsonSchema
+          location: ./real-jsonschema.json
+
+Combination of OAS schema and alternative:
+
+    schema:
+        type: object
+        nullable: true
+        x-oas-draft-alternativeSchema:
+            type: jsonSchema
+            location: ./real-jsonschema.json
+
+Multiple different versions of alternative schema:
+
+    schema:
+        anyOf:
+            - x-oas-draft-alternativeSchema:
+                type: jsonSchema
+                location: ./real-jsonschema-08.json
+            - x-oas-draft-alternativeSchema:
+                type: jsonSchema
+                location: ./real-jsonschema-07.json
+
+Combined alternative schemas:
+
+    schema:
+        allOf:
+            - x-oas-draft-alternativeSchema:
+                type: xmlSchema
+                location: ./xmlSchema.xsd
```

Comment:
@cmheazel      @andreapoli

How to support entities in an xsd? Would this work?

```
           type: xmlSchema
           location: schema.xsd#/person
```

Reply:
Whole files only I believe, as it would be tough to support XML Path, JSONPath and Protobufpath wait there is no protobuf path. Better not use paths. :)

Reply:
imho `location` value could support path only for some `type`. The `type` could dictate the `location` patterns.